README.md

```diff
@@ -3,7 +3,8 @@ JSON-Patch
 
 > A leaner and meaner implementation of JSON-Patch. Small footprint. High performance.
 
-[![Build Status](https://travis-ci.org/Starcounter-Jack/JSON-Patch.svg?branch=master)](https://travis-ci.org/Starcounter-Jack/JSON-Patch)
+[![Build
+Status](https://travis-ci.org/Schrodinger/JSON-Patch.svg?branch=master)](https://travis-ci.org/Schrodinger/JSON-Patch)
```

Comment:
Not strictly required.. but it's a little misleading to show a "Passing" badge from the original repo. The builds on our branch are actually failing, but I haven't looked into fixing / updating them because we may not continue to use this fork much longer.

If we do decide to keep this fork in use, we should definitely hook up Travis.

Reply:
haha definitely.  thanks!